### ISSUE TITLE
Divide grids into # of circles or intervals by an array of strings

### ISSUE THREAD

**Issue by alex-w** [Stellarium/stellarium#370]
_Original report by John Carlson:_ https://bugs.launchpad.net/stellarium/+bug/1722425

I would like to divide the various grids into a configurable number of intervals, not just a boolean on/off, and then label them, like "Northern Hemisphere", "Western Hemisphere", or the months of the year (putting ranges of dates into buckets spread across 365 or 366 days), similar to how the 24 intervals are shown now. If this can be done with a plugin, please point at n plugin or provide simple code which can be extended. Something better than boolean on (24 circles or intervals) / off (0 circles or intervals) for grids should be provided. I suggest a # circles * interval in degrees = 360 degrees. I suspect that # of circles would be a better parameter than interval in degrees, but I don't really know. Perhaps instead # of circles, one could pass an ordered list of strings of labels, the number of strings being the number of circles or intervals.

Basically, take an array (possibly 2 dimensional) of strings (all hemispheres, for example, or days in a year) and distribute them into a grid.

Furthermore, allowing the array of strings to be mapped to other arrays of strings, hyperlinks, SVGs or image icons would be even more desirable, for possible display, translation or mapping purposes.

John Carlson

**Comment by gzotti** [comment]
Not related to #993. OP wants manual grid width steps instead of the automatism which is now in place decreases steps when zooming in.
